test/components/checkbox.spec.tsx

```diff
@@ -160,23 +160,62 @@ describe('<Checkbox/>', function() {
 
         return waitFor(() => {
             expect(onChange).to.have.been.calledOnce;
-            expect(onChange).to.have.been.calledWith(false);
+            expect(onChange).to.have.been.calledWithMatch({value: false});
         });
     });
 
-    it('Renders a native input component of type "checkbox" for SEO purposes', function() {
-        const {select} = clientRenderer.render(
-            <CheckBox
-                boxIcon={boxSVG}
-                tickIcon={tickSVG}
-                value={true}
-            />
-        );
-        const nativeInput = select('NATIVE_CHECKBOX') as HTMLInputElement;
+    describe('Accessibility features', function() {
+        it('Renders a native input and pass on checked state', function() {
+            const {select, waitForDom} = clientRenderer.render(
+                <CheckBox value={true}/>
+            );
+            const nativeInput = select('NATIVE_CHECKBOX') as HTMLInputElement;
+
+            return waitForDom(() => {
+                expect(nativeInput, 'native input not found in DOM').to.exist;
+                expect(nativeInput.tagName).to.equal('INPUT');
+                expect(nativeInput).to.have.attr('type', 'checkbox');
+                expect(nativeInput.checked, 'native checkbox should be checked').to.equal(true);
```

Comment:
the `.checked` attribute doesn't appear in the dom so the `.attribute` matcher doesn't work the same way